http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HeadersIT.java

```diff
@@ -93,6 +96,19 @@ void testPathSpecificHeaderRulesOrder() {
         cacheControlMatches(response, "max-age=1");
     }
 
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/pull/41411")
```

Comment:
I refuse to believe that PR only changing independent RESTEasy Reactive library has to be tested in the RESTEasy Classic. They are completely independent implementations.

If you mean to test that same issue is not present in the RESTEasy Classic, I can respect that (though I would not), but I would not link the PR. Maybe link the issue and add comment that this is not reproducing, just verifying it is not there?

Reply:
TBH I was thinking yesterday if I should remove this classic coverage. Your reasoning make sense so I remove it.

@jcarranzan if you still think that we should test even the rest classic please create PR

Reply:
it's nice you be agreed with @jedla97 , but @rsvoboda doesn't :-) maybe just tweak the code.

my thinking was that users shouldn't use RESTEasy Classic anyway, so I care way less about it.

Reply:
Oh now I read @rsvoboda comment so I will update it back

Reply:
Reverted back + changed to use original issue in comment